app/views/handlers/question.py

```diff
@@ -101,3 +104,13 @@ def _render_block(self, block_id):
         )
 
         return {**transformed_block, **{'question': rendered_question}}
+
+    def are_hub_required_sections_complete(self):
```

Comment:
As discussed, this would make sense to live in router.py:
```py
    def can_access_hub(self):
        if self._schema.is_hub_enabled() and all(
            self._progress_store.is_section_complete(section_id)
            for section_id in self._schema.get_section_ids_required_for_hub()
        ):
            return True

        return False
```

Reply:
thanks, I moved it to router.py